### DISCUSSION TITLE
《开源之迷》书评 | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#18] [Comments]
# 《开源之迷》书评 | 夜天之书

《开源之迷》是“开源之道”主创适兕所著“开源之道三部曲”的第一部。本书的“迷”是着迷的“迷”，旨在向读者介绍什么是开源，以及开源令人欲罢不能的优势。适兕在《适兕是如何生存的？》提到，自己从 2015 开始运行“开源之道”，2018 年辞去全职工作，一心发展开源之道。开源之迷，一至于此。 本书发售前，我曾经应邀写了一小段推荐词，不过最后因为出版的原因没有随书印刷。这里附上原文内容，也可以对比只读样章

https://tisonkun.org/2022/04/21/the-fascination-of-open-source/

**Comment by donhui** [comment]
推荐序中的《开源之谜》应该改为《开源之迷》吧？抑或是作者有意写错？

**Reply by tisonkun** [reply]
麻了，应该是当时写的时候写错了。
